### ISSUE TITLE
Add BlTouch

### ISSUE THREAD

**Issue by dj-fiorex** [NightHammer1000/Marlin_ThinkerSE#3]
### Description

Hi, what steps should I do if I want to add support for the BlTouch sensor?

**Comment by NightHammer1000** [comment]
I dont own a BLTouch. Never did.
But I plan to in the next 2 Months.
A Config will be added then.

**Comment by dj-fiorex** [comment]
I am a developer, can i help you with the configuration? I have a BlTouch V3.1

**Comment by NightHammer1000** [comment]
Yeah.
Everything we need for it is already there.
You just need to enable BLTouch and UBL in the Config and find the Right Values to tune it

**Comment by dj-fiorex** [comment]
Ok perfect, i think the best way is to get a look at the configuration with BlTouch of the original Eryone repo, and make the appropriate changes to this repo configuration file

**Comment by NightHammer1000** [comment]
Yeah. Just please use UBL and not any of the Other Auto Bed Leveling Solutions.

UBL Produces way better Results.

You can send a pull request once its done. I will merge it.

**Comment by dj-fiorex** [comment]
Hello all, just a little problem, today i tried the first version of your firmware with my changes about UBL and BLTouch. The problem is that i flashed the firmware successfully (after disconnect the display) but after turning off, reattach the screen, turning on my display is totally blank, what happened?
